target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-lenovo-ix2-dl.dts

```diff
@@ -0,0 +1,48 @@
+/* Device tree file for the Lenovo IOMEGA IX2-dl NAS box.
+ *
+ * Copyright (c) 2017, Daniel Helgason <daniel.k.helgason @ gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version
+ * 2 of the License, or (at your option) any later version.
+ *
+ * Based on the device-tree work of many fine linux developers.
+ */
+
+/dts-v1/;
+
+#include "kirkwood-lenovo-ix2-common.dtsi"
+
+/ {
+	model = "Lenovo IX2-dl";
+	compatible = "lenovo-ix2-dl", "marvell,kirkwood-88f6282", "marvell,kirkwood";
+
+};
+
+&nand {
+	status = "okay";
+	chip-delay = <35>;
+
+	partition@0 {
+		label = "uboot";
+		reg = <0x0000000 0x0080000>;
```

Comment:
Please use the same number of digits everywhere or remove all leading zeros.

First offset should be just 0x0